bare/main.go

```diff
@@ -46,11 +49,23 @@ func main() {
 		{config: "bare/config/runner.jsonnet", binary: "bb_runner"},
 		{config: "bare/config/browser.jsonnet", binary: "bb_browser"},
 	}
+	var commands []*exec.Cmd
 	for _, bb := range bbs {
 		log.Println("Starting", bb.binary)
-		go bbStart(bb)
+		commands = append(commands, bbStart(bb))
 	}
 
+	// Kill all started processes if interrupted:
+	c := make(chan os.Signal)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
+		for _, command := range commands {
+			log.Printf("Killing proc: %d", command.Process.Pid)
+			command.Process.Kill()
+		}
+		os.Exit(1)
```

Comment:
Nit: Instead of doing an `os.Exit(1)` here, shouldn't we re-raise the `SIGTERM`?

Reply:
I implemented the suggestion from @moroten and there no longer is any point in re-raising the `SIGTERM` since the application just finishes after stopping all the started processes. Though that means the return code is `0`